examples/pass-parameters-to-tests/README.md

```diff
@@ -0,0 +1,23 @@
+# Pass Parameters to Tests
+
+**Test Code**: [index.js](index.js)
+
+This example demonstrates how to pass parameters to TestCafe tests from the following sources:
+
+* a JSON configuration file
+* a JavaScript object
+* an environment variable
+
+Before the tests, the `.json` and `.js` configuration files are imported with JavaScript [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) syntax. The environment variable is declared with the [process.env](https://nodejs.org/api/process.html#process_process_env) property. These values are then available inside tests.
+
+During each of the tests, the `test.page` method reads one of these values to decide the page where the test starts. A `clientFunction` that is declared before the tests begin obtains the full URL of the current page. The `t.expect.eql` method compares this value to the full path to the file that the configuration file or the environment variable specified.
```

Comment:
For each test, the initial page is set by calling to the [test.page](https://devexpress.github.io/testcafe/documentation/reference/test-api/test/page.html) method with appropriate parameter.

Reply:
> For each test, a `test.page` method call with an appropriate parameter sets the initial page. A `clientFunction` that is declared before the tests begin obtains the full URL of the current page. The `t.expect.eql` method compares this value to the full path to the file that the configuration file or the environment variable specified.